specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/billingSubscription.json

```diff
@@ -612,131 +681,545 @@
+        "invoiceSectionName": {
+          "description": "The name of the invoice section to which the subscription is billed. The field is applicable only for Microsoft Partner Agreement billing accounts.",
           "type": "string",
-          "enum": [
-            "Active",
-            "Inactive",
-            "Abandoned",
-            "Deleted",
-            "Warning"
-          ],
-          "x-ms-enum": {
-            "name": "BillingSubscriptionStatusType",
-            "modelAsString": true
-          }
+          "readOnly": true
         },
         "lastMonthCharges": {
-          "description": "The last month charges.",
+          "description": "The last month charges. This field is only available for consumption subscriptions of Microsoft Customer Agreement billing accounts.",
           "readOnly": true,
           "$ref": "#/definitions/Amount"
         },
         "monthToDateCharges": {
-          "description": "The current month to date charges.",
+          "description": "The current month to date charges. This field is only available for consumption subscriptions of Microsoft Customer Agreement billing accounts.",
           "readOnly": true,
           "$ref": "#/definitions/Amount"
         },
-        "billingProfileId": {
-          "description": "The ID of the billing profile to which the subscription is billed.",
-          "type": "string",
-          "readOnly": true
+        "nextBillingCycleDetails": {
+          "description": "Next billing cycle details of the subscription.",
+          "type": "object",
+          "$ref": "#/definitions/NextBillingCycleDetails"
         },
-        "billingProfileDisplayName": {
-          "description": "The name of the billing profile to which the subscription is billed.",
+        "offerId": {
+          "description": "The offer ID for the subscription. This field is only available for the Microsoft Online Services Program billing accounts.",
           "type": "string",
           "readOnly": true
         },
-        "costCenter": {
-          "description": "The cost center applied to the subscription.",
-          "type": "string"
-        },
-        "customerId": {
-          "description": "The ID of the customer for whom the subscription was created. The field is applicable only for Microsoft Partner Agreement billing account.",
+        "productCategory": {
+          "description": "Category of the subscription. Possible values include: AzureSupport, Hardware, ReservationOrder, SaaS, SavingsPlanOrder, Software, UsageBased, Other",
           "type": "string",
           "readOnly": true
         },
-        "customerDisplayName": {
-          "description": "The name of the customer for whom the subscription was created. The field is applicable only for Microsoft Partner Agreement billing account.",
+        "productType": {
+          "description": "Type of the product for which the subscription is purchased",
```

Comment:
Just to be consistent with other descriptions let's add a "The" in the beginning. "The type of the product for which ...."